src/index.ts

```diff
@@ -6,7 +6,8 @@ function arrayMustNotBeEmpty(value: Array<any>): boolean {
 }
 
 function emailMustBeValid(value: string): boolean {
-  return /.+@.+/.test(value);
+  const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
Add a comment: this regex comes from http://emailregex.com/ and will match 99.99% of modern email addresses